src/main/java/edu/kis/powp/command/OperateToCommand.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.command;
+
+public class OperateToCommand {
+    public void operateTo(int x, int y) {
```

Comment:
Subcommands classes should implement DriverCommand. On diagram x and y are properties (fields) not params of any method.